plugins/backstage-plugin-coder/src/components/CoderProvider/CoderAuthProvider.tsx

```diff
@@ -152,50 +140,140 @@ export const CoderAuthProvider = ({ children }: CoderAuthProviderProps) => {
   // outside React because we let the user connect their own queryClient
   const queryClient = useQueryClient();
   useEffect(() => {
```

Comment:
Effect logic is exactly the same as before; just refactored it slightly for readability